style.css

```diff
@@ -0,0 +1,155 @@
+html {
+    font-size: 10px;
+    font-family: 'Lato', sans-serif;
+    scroll-behavior: smooth;
+    box-sizing: border-box;
+}
+h1 {
+    color: #ffffff;
+    font-weight: 700;
+    font-size: 10rem;
+    text-align: center;
+}
+h2 {
+    font-size: 4rem;
+    margin: 50px 0;
+    text-align: center;
+}
+h3 {
+    font-size: 2rem;
+    margin: 0;
+    text-align: center;
+}
+p {
+    font-size: 1.5rem;
+}
+header {
+    background-color: #001453;
+    padding: 50px 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    color: #ffffff;
+}
+.new-item-form-section {
+    padding: 50px;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    gap: 25px;
+}
+form {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+    gap: 30px;
+    max-width: 600px;
+}
+form input, select {
```

Comment:
an `input` should always be inside a `form` anyway if we're following best practice